haiku-apps/commandtimer/commandtimer-2.0~git.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Run a command after a specified amount of time has passed"
```

Comment:
I think _"after a specified **amount of time** has passed"_ does not sound very nice.
What do you think of:
```
SUMMARY="Launch or repeat commands after a user-chosen count down"
```